index.html

```diff
@@ -1,11 +1,582 @@
+      </div>
+    </section> <!-- End Schedule Section -->
+    <!-- ======= Register Section ======= -->
+    <section id="register">
+      <div class="container" data-aos="zoom-in">
+        <div class="section-header-inv">
+          <h2>Register</h2>
+          <!-- <p>Join our Slack!</p> -->
+          <!-- <p>Please read our <a style="color:white; text-decoration: underline;" href="rules.html">Eligibility & Rules</a> before registering for the competition.</p> -->
+        </div>
+        <center>
+          <p style="width: 80%">
+            To register, you must be at least 13 years old <b>and</b> in high school. Register now through the Google Forms, or click the button.
+          </p>
+          <button type="button" onclick="window.open('register.html')">Register Now!</button></center><br>
+        </center>
+        <div id="googleForm">
+          <iframe src="https://docs.google.com/forms/d/e/1FAIpQLScOwx6NrWSfE5pQUDSLjpszBgvs9jQeLw84XFpZAawVHjm6mA/viewform?embedded=true" width="640" height="750" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
+        </div>
+      </div>
+    </section><!-- End Register Section -->
+
+    <!-- =======  F.A.Q Section ======= -->
+    <section id="faq">
+
+      <div class="container" data-aos="fade-up">
+
+        <div class="section-header">
+          <h2>Frequently Asked Questions</h2>
+        </div>
+
+        <div class="row justify-content-center" data-aos="fade-up" data-aos-delay="100">
+          <div class="col-lg-9">
+            <ul id="faq-list">
+              <!-- <li>
+                <a data-toggle="collapse" href="#faq2" class="collapsed">How do the organizers pick the tasks?<i class="fa fa-minus-circle"></i></a>
+                <div id="faq2" class="collapse" data-parent="#faq-list">
+                  <p>
+                    The tasks were chosen to serve the real time needs of the organization. Thus, your efforts during the competition will directly contribute to the success of PathCheck’s mission to reopen society. 
+                  </p>
+                </div>
+              </li> -->
+
+              <li>
+                <a data-toggle="collapse" href="#faq3" class="collapsed">I will be turning 13 after the beginning of the event, can I still participate? <i class="fa fa-minus-circle"></i></a>
+                <div id="faq3" class="collapse" data-parent="#faq-list">
+                  <p>
+                    Unfortunately, we are only allowing participants who are within the eligible age range on the day of the event. 
+                  </p>
+                </div>
+              </li>
+
+              <li>
+                <a data-toggle="collapse" href="#faq4" class="collapsed">Do I get a certificate? <i class="fa fa-minus-circle"></i></a>
+                <div id="faq4" class="collapse" data-parent="#faq-list">
+                  <p>
+                    MIT Professor Ramesh Raskar will be signing digital certificates for approved essays.
```

Comment:
All authors whose essays are published on the MIT Pathfinders website will receive a digitally signed certificate from MIT Media Lab Professor Ramesh Raskar (lin to his site)